iOS/Source/SunView.swift

```diff
@@ -176,10 +170,15 @@ class SunView: UIView {
         animation.delegate = self
 
         self.sun.layer.add(animation, forKey: "animate position along path")
-        self.sunViewLocation = self.location(for: CGFloat(percentage))
+        let newLocation = self.location(for: CGFloat(percentage))
+        self.sunViewLocation = newLocation
     }
 
     func updateInterface(withBackgroundColor backgroundColor: UIColor, textColor: UIColor, andPercentageInDay percentageInDay: Double, sunPhase: SunPhase) {
+        self.currentTimeLabel.text = self.timeFormatter.string(from: Date())
+        self.sunriseLabel.text = Location.current?.sunriseTimeString
+        self.sunsetLabel.text = Location.current?.sunsetTimeString
```

Comment:
I think is better if we inject the location here instead of assuming that there's a `Location.current`.  Because keeping assumptions at a minimum makes software easier to test and to maintain.